### PR TITLE
Add Leaderboard Site

### PR DESCRIPTION
**Pull request by sampoder**
(also has my one other change)

### CODE REVIEW

File: public/leaderboard.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
```
Comment (lachlanjc) [inline public/leaderboard.html:1]: Protip: make this leaderboard/index.html & you won’t need the rewrite!

File: components/nav.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ const Join = () => {
           margin-left: 16px;
           text-decoration: none;
           text-transform: uppercase;
+          color: #fff;
```
Comment (lachlanjc) [inline components/nav.js:18]: Need to edit this & it’s in a separate PR…can we remove from here?

Comment (sampoder) [reply components/nav.js:18]: sure thing

File: public/leaderboard.html (hunk 1 above)
Comment (sampoder) [reply public/leaderboard.html:1]: ahhh i thought there was a way

**Comment by lachlanjc** [comment]
762D576F-17D0-4C42-A4DE-D1153281CDC2

Happy to merge without but FYI mobile could use some love

**Comment by sampoder** [comment]
mobile has it's own table that only mobile people can see (and same for desktop so that shouldn't be too hard to fix)

**Pull request merged**